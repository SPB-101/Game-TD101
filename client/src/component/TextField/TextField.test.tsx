import React from "react";
import { shallow } from "enzyme";
import { TextField } from "./index";

describe("<TextField />", () => {
  it("TextField renders correctly", () => {
    const wrapper = shallow(<TextField name="login" />);
    expect(wrapper).toMatchSnapshot();
  });
});
