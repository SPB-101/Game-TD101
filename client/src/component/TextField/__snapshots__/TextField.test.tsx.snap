// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextField /> TextField renders correctly 1`] = `
<label
  className="text-field"
  htmlFor="login"
>
  <input
    className="text-field__input"
    name="login"
  />
  <div
    className="text-field__label"
  />
  <span
    className="text-field__error"
  />
</label>
`;
